beamerinnerthemeOSU.sty

```diff
@@ -27,7 +27,7 @@
                 \node[text centered, text width=10cm] (author) at (\paperwidth/2, 3) {\usebeamerfont{author}\insertauthor\\ \medskip\usebeamerfont{date}\insertdate};
             \else
                 \node (graphic) at (\paperwidth/2, \paperheight/2) {\inserttitlegraphic};
-                \node[text centered, text width=14cm, osu_orange] (title) at (\paperwidth/2, 8) {\usebeamerfont{title}\inserttitle\\ \medskip\usebeamerfont{subtitle}\textcolor{osu_black}{\insertsubtitle}};
+                \node[text centered, text width=14cm, osu_orange] (title) at (\paperwidth/2, 7) {\usebeamerfont{title}\inserttitle\\ \medskip\usebeamerfont{subtitle}\textcolor{osu_black}{\insertsubtitle}};
```

Comment:
This line may need to be reverted to position the title at `(\paperwidth/2, 8)`. As can be seen [here](https://user-images.githubusercontent.com/56184047/84149558-83c54480-aa15-11ea-8dd8-b27904424cd3.png), the subtitle can be cut off depending on the size of the `\titlegraphic`.

Unless you know of a way to detect the size off the title graphic automatically and position the title accordingly, the user will have to manually set this each time. I've added instructions on how to do this to my README.

I suggest using `(\paperwidth/2, 8)` since this effectively aligns the title as close to the top of the frame as can still be considered aesthetically pleasing, while maximizing the space available for the `\titlegraphic` (minimizing the chances the user will have to get involved in the `.sty` files at all). Very open to other suggestions though!